packages/firestore/src/api/database.ts

```diff
@@ -922,8 +922,7 @@ export class DocumentReference implements firestore.DocumentReference {
     }
 
     const internalOptions = {
-      includeDocumentMetadataChanges: options.includeMetadataChanges,
-      includeQueryMetadataChanges: options.includeMetadataChanges
+      includeMetadataChanges: options.includeMetadataChanges
```

Comment:
FWIW Gil / Rich didn't actually change the internal ListenOptions class... so TypeScript has diverged...  but simplifying it seems like an improvement and not a particularly confusing divergence, so it's probably fine for now.

Reply:
Unless we plan need to differentiate between these states, I would also argue that this is an improvement (a slightly biased argument I might say).